src/test/kotlin/org/opensearch/observability/rest/AssemblyValidationIT.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.observability.rest
+
+import org.junit.Assert
+import org.opensearch.observability.PluginRestTestCase
+import org.opensearch.rest.RestRequest
+import org.opensearch.rest.RestStatus
+
+class AssemblyValidationIT : PluginRestTestCase() {
+    companion object {
+        private const val TRACES_MAPPING_TEMPLATE_NAME = "sso_trace_template"
+        private const val METRICS_MAPPING_TEMPLATE_NAME = "sso_metric_template"
+    }
+
+    fun `test observability traces template and was created`() {
+        // verify traces mapping template was created successfully as part of the plugin initialization
+        Thread.sleep(1000)
```

Comment:
is there a better way for this? Not familiar with backend testing...

Reply:
I'm not familiar with other way of getting client notified of cluster ready ...
This is common in our code base